addons/hr_expense/models/hr_expense.py

```diff
@@ -523,15 +523,14 @@ def refuse_expense(self, reason):
                                              values={'reason': reason, 'is_sheet': False, 'name': self.name})
 
     @api.model
-    def get_expense_dashbord(self):
-        if not self.env.user.employee_ids:
-            return
-        expenses = self.read_group(
-            [
-                ('employee_id', 'in', self.env.user.employee_ids.ids),
-                ('payment_mode', '=', 'own_account'),
-                ('state', 'in', ['draft', 'reported', 'approved'])
-            ], ['total_amount', 'currency_id', 'state'], ['state', 'currency_id'], lazy=False)
+    def get_default_dashboard(self, expense_state):
```

Comment:
You can't change a method signature in a stable release ;)

See: https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean

Reply:
Already fixed btw.